rust/cardano-blockchain-types/src/utils/mod.rs

```diff
@@ -0,0 +1,3 @@
+//! Utility functions.
```

Comment:
Make a new public library crate `cbork-util` move this module to it.
For now, its all that `cbork-util` will contain.